geoengine/datasets.py

```diff
@@ -331,8 +331,8 @@ def to_api_dict(self) -> api.UploadId:
 
 class DatasetProperties():
     '''The properties of a dataset'''
-    id: Optional[DatasetId]
-    name: str
+    name: Optional[str]
```

Comment:
it's only optional for AddDatasetProperties right?

Reply:
[0d2d3dd](https://github.com/geo-engine/geoengine-python/pull/128/commits/0d2d3dd1e1ac45ac0860bad84087d4dc90b67fca)